src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/SmsSecondFactor/index.tsx

```diff
@@ -129,66 +138,81 @@ export const SmsSecondFactor: React.FC<SmsSecondFactorProps> = ({
     })
   }
 
+  const show2FAWarning = isArtsyEmail(me.email!)
+
   return (
     <>
-      <Flex
-        p={2}
-        border="1px solid"
-        borderColor="black10"
-        flexDirection={["column", "row"]}
-      >
-        <Box flexBasis="50%">
-          <Text variant="lg-display">Use Text Messages</Text>
-
-          {enabledSecondFactorLabel && (
-            <Text variant="lg-display" color="black60">
-              {enabledSecondFactorLabel}
+      <Flex flexDirection="column" border="1px solid" borderColor="black10">
+        {show2FAWarning && (
+          <Message variant="warning">
```

Comment:
**suggestion**: If the condition to show this copy is only that an admin is logged in, maybe we drop the severity level to `info`?

Reply:
We might also consider hiding this message if a user is already enrolled with an App Authenticator method.

Reply:
Actually, I take this suggestion back! As this message is tightly scoped to the SMS enrollment section, a warning message makes sense 👍🏼